assets/src/dashboard/app/views/myStories/myStoriesSearch.js

```diff
@@ -58,15 +58,11 @@ export default function MyStoriesSearch({
 
 MyStoriesSearch.propTypes = {
   handleChange: PropTypes.func.isRequired,
-  currentValue: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  currentValue: PropTypes.oneOfType([PropTypes.string]),
```

Comment:
Super nit, can this just be `currentValue: PropTypes.string`?

Reply:
yes :) thank you